src/IonBinaryWriter.ts

```diff
@@ -311,10 +311,9 @@ export class BinaryWriter extends AbstractWriter {
   }
 
   private encodeAnnotations(annotations: string[]) : Uint8Array {
-    if (!annotations || annotations.length === 0) {
+    if (annotations.length === 0) {
```

Comment:
Since this is a private method I'm assuming the invariant here is that annotations is never `null` nor `undefined`, is this correct? 

If it is add a comment about it

Reply:
yes, inputs are enforced or set internally.